src/renderer/viz/expressions/Animation.js

```diff
@@ -49,6 +49,9 @@ let waitingForOthers = new Set();
  *   color: ramp(linear(clusterAvg($temp), 0,30), tealrose)
  *   filter: animation(linear($date, time('2022-03-09T00:00:00Z'), time('2033-08-12T00:00:00Z')), 40, fade(0.1, 0.3))
  * `);
+ * Animation class
+ *
+ * This class is instanced automatically by using the `animation` function. It is documented for its methods.
```

Comment:
I'd change the phrasing. I'd say something like: the `animation` function returns instances of the `Animation` class